lib/puppet/type/file/selcontext.rb

```diff
@@ -58,6 +63,18 @@ def insync?(value)
       end
     end
 
+    def unsafe_munge(should)
+      if not selinux_support?
+        should
```

Comment:
It may not be very ruby-like but given how many implicit returns are in these methods, maybe it would be more readable to do explicit `return should` etc.?